Rakefile

```diff
@@ -1,52 +1,123 @@
-#  IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-########################################################################################################################
+require 'bundler'
+Bundler.setup
 
-require 'bundler/gem_tasks'
 require 'rspec/core/rake_task'
 
-RSpec::Core::RakeTask.new(:spec)
+# Always create spec reports
+require 'ci/reporter/rake/rspec'
+
+# Gem tasks
+require 'bundler/gem_tasks'
+
+RSpec::Core::RakeTask.new('spec:unit') do |spec|
+  spec.rspec_opts = %w[--format progress --format CI::Reporter::RSpec]
+  spec.pattern = FileList['spec/**/*_spec.rb']
+end
 
-# Load in the rake tasks from the base openstudio-extension gem
-# Also set measures_dir location for this gem for openstudio-extension gem to use
-require 'openstudio/extension/rake_task'
-require 'buildingsync'
-rake_task = OpenStudio::Extension::RakeTask.new
-rake_task.set_extension_class(BuildingSync::BuildingSync)
+task 'spec:unit' => 'ci:setup:rspec'
+task 'spec:unit' => 'init_config'
+task default: 'spec:unit'
 
 require 'rubocop/rake_task'
-RuboCop::RakeTask.new
+desc 'Run RuboCop on the lib directory'
+RuboCop::RakeTask.new(:rubocop) do |task|
+  task.options = ['--no-color', '--out=rubocop-results.xml']
+  task.formatters = ['RuboCop::Formatter::CheckstyleFormatter']
+  task.requires = ['rubocop/formatter/checkstyle_formatter']
+  # don't abort rake on failure
+  task.fail_on_error = false
+end
+
+desc 'Update copyright on files'
+task :update_copyright do
+  require 'fileutils'
+  
+  basepath = File.dirname(__FILE__)
+  
+  ruby_copyright = "########################################################################################################################\n"
+  File.open(basepath + "/LICENSE.md") do |f|
+    while (line = f.gets)
+      if line.strip.empty?
+        ruby_copyright += "#" + line
+      else
+        ruby_copyright += "#  " + line
+      end
+    end
+  end
+  ruby_copyright += "########################################################################################################################\n\n"
+  
+  # files that are part of BRICR
+  paths = [basepath + "/bin/**/*.rb",
+           basepath + "/lib/**/*.rb",
+           basepath + "/*.rb"]
+           
+  paths.each do |path|
+    # glob for rb
+    files = Dir.glob(path)
+    files.each do |file|
+      next if File.basename(file) == 'config.rb'
+      
+      # start with copyright
+      text = ruby_copyright
+
+      # read file
+      File.open(file, "r") do |f|
+        # read until end of current copyright
+        while (line = f.gets)
+          if not /^#/.match(line)
+            if not line.chomp.empty?
+              text += line
+            end
+            break
+          end
+        end
+
+        # now keep rest of file
+        while (line = f.gets)
+          text += line
+        end
+      end
+
+      # write file
+      File.open(file, "w") do |f|
+        f << text
+      end
+  
+    end
+  end
+
+  puts ruby_copyright
+end
+
+
+desc 'Init config'
```

Comment:
I don't think we need this as I don't think we should be spinning up a SEED instance in the tests, @nllong is that right?  We should disable the Travis tests in preference for the Jenkins ones that @tijcolem has set up.  Basically, there are predefined Jenkins tests for all extension gems, we can update them in one place.